backend/app/tests/crud/test_inquiry.py

```diff
@@ -1,10 +1,11 @@
 import random
 import string
 
+from fastapi.encoders import jsonable_encoder
```

Comment:
Why are test_inquiry and test_inquiries in a separate file

Reply:
* `test_inquiries` is responsible for API layer and checks if API returns correct payload and status code.
* `test_inquiry` is responsible for DB layer and checks if DB persists with the changes as we intended.

Reply:
yes.  you are right.  sorry i missed that.